isis/tests/FunctionalTestsPointreg.cpp

```diff
@@ -0,0 +1,377 @@
+#include "pointreg.h"
+
+#include "Fixtures.h"
+#include "TestUtilities.h"
+#include "UserInterface.h"
+#include "ControlNet.h"
+#include "LineManager.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/pointreg.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestPointregDefault) {
+  // Populate cubes
+  LineManager line(*cube1);
+  double pixelValue = 0.0;
+  for(line.begin(); !line.end(); line++) {
+    for(int i = 0; i < line.size(); i++) {
+      line[i] = pixelValue++;
+    }
+    cube1->write(line);
+    cube2->write(line);
+    cube3->write(line);
+  }
+  cube1->close();
+  cube2->close();
+  cube3->close();
+
+  Pvl log;
+  QTemporaryDir prefix;
+  QString flatFilePath = prefix.path() + "/flatfile.csv";
+  QString outNetPath = prefix.path() + "/outNet.net";
+  QVector<QString> args = { "fromlist=" + cubeListFile,
+                            "cnet=data/threeImageNetwork/controlnetwork.net",
+                            "deffile=data/threeImageNetwork/autoRegTemplate.def",
+                            "flatfile=" + flatFilePath,
+                            "onet=" + outNetPath };
```

Comment:
I think the fixture has a var for the cnet path, I would use that and add the template to a variable as well. It'll make things easier to move around in the future.